frontend/homework/8/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Recipie shop</title>
+        <link href="./style.css" rel="stylesheet">
+    </head>
+    <body>
+        <div class="main-dev">
+            <div class="title">
+                <p class="title-text"> Recipe shop</p>
+            </div>
+            <form id="search">
+            <div class="search-div">
+                <input type="text" id="text-area" placeholder="search">
+                <button type="submit" id="submit-btn">submit</button>
+            </div>
+            </form>
+            <div class="data-div">
+            <div class="dec-div" id="recipeList">
+
+            </div>
+        </div>
+        </div>
+        <script src="./app.js" ></script>
+      <script>
```

Comment:
Avoid inline scripts.
Set `allowJs=true `in config and manipulate accordingly for TS-JS inter-communication